src/test/java/core/basesyntax/handlers/BalanceHandlerTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.BalanceStrategy;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    @Test
+    public void handle_ValidTransaction_ShouldUpdateBalance_Ok() {
+        BalanceStrategy balanceStrategy = new BalanceStrategy();
+        FruitTransaction transaction = new FruitTransaction(FruitTransaction
+                .Operation.BALANCE, "banana", 100);
+        Map<String, Integer> fruitStore = new HashMap<>();
+        balanceStrategy.handleTransaction(transaction, fruitStore);
+
+        int expectedQuantity = 100;
+        int actualQuantity = fruitStore.get("banana");
+
+        assertEquals(expectedQuantity, actualQuantity);
```

Comment:
if you split parts of the unit test by empty line - do it properly. Use this pattern for your tests Arrange, Act, Assert
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80